src/commands/docs/generateDot.ts

```diff
@@ -0,0 +1,40 @@
+import { getConstants } from '../../constants'
+
+import { getLocalConfig } from '../../utils/config'
+import { getFoldersForDocs } from './internal/getFoldersForDocs'
+import { createDotFiles } from './internal/createDotFiles'
+import { getDocConfig } from './internal/getDocConfig'
+
+/**
+ * Generates lineage in dot language
+ * By default the dot files will be at 'sasjsbuild/docs' folder
+ * If a target is supplied, generates dot files only for the jobs / services in that target (and the root).
+ * If no target is supplied, generates for all jobs / services.
```

Comment:
I think this comment is not valid any more

Reply:
Yes, I need to update/add comments and tests for `sasjs doc lineage`